src/node.cc

```diff
@@ -3307,6 +3381,11 @@ void LoadEnvironment(Environment* env) {
   // Add a reference to the global object
   Local<Object> global = env->context()->Global();
 
+  Local<Object> JSArray = global->Get(env->array_class_string()).As<Object>();
+  Local<Function> JSFrom =
+      JSArray->Get(env->array_from_string()).As<Function>();
```

Comment:
If you're only using these strings once, there is not much point in adding them to the env.  Just create them with `FIXED_ONE_BYTE_STRING()` here.

Also, the variable names are minor style violations; they should be called (e.g.) `js_array_object` and `js_array_from_function`.